src/components/toolbar/index.js

```diff
@@ -25,6 +25,33 @@ class Toolbar extends Component {
     super(props)
   }
 
+  handleShortcuts = event => {
+    const { toggleSetting, navigateToPage } = this.context
+    const { isFirstPage, isLastPage, currentPage } = this.context.state
+
+    switch (event.keyCode) {
+      case 70: //Key: f
+        toggleSetting('fullscreen')
+        break
+
+      case 39: //Key: Right Arrow
+        if (!isLastPage) navigateToPage(currentPage + 1)
```

Comment:
We use arrow keys to control the slider component, can you check if there are any conflicts ?

Reply:
The slider component works consistently while using the arrow keys.
![slider](https://user-images.githubusercontent.com/25086429/67163411-20a27700-f38c-11e9-96c9-d1df3e6b4f81.gif)

Reply:
oh, they are actually doing the same thing lol :stuck_out_tongue:  my bad,